modules/API/IMUController.h

```diff
@@ -91,6 +104,7 @@ class IMUController {
   mutable std::mutex imuMutex;  // IMUデータのスレッドセーフなアクセス用ミューテックス
   std::thread angleCalculationThread;  // 角度計算用のスレッド
   bool isCalculating = false;          // 角度計算実行中フラグ
+  bool startedByCommand;  // コマンドによって開始されたかを示すフラグ
```

Comment:
この変数名だと何をするためのものかあんまり分かりにくかったです
でも、改善案でしっくりくるのがない...

せめて、コメントでは「何が」開始されたかを説明してほしい
あと、（フラグを示すような）bool値の変数は「is」つけてくれると分かりやすい
```suggestion
  bool isStartedByCommand;  // 角度計算がコマンドによって開始されたかを示すフラグ
```

逆に、何のためのフラグかを示すようにするとか？
```suggestion
  bool isCheckingSelfAngle;  // 自己角度観測中かを示すフラグ
```

Reply:
追加で言うと、コマンドによって実行されたかはこのクラスからじゃわからないので、ByCommandいるかな？とも思う。直接呼んじゃってもいいわけだし、可能なわけだし。